pom.xml

```diff
@@ -238,6 +238,14 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <version>3.1.11</version>
```

Comment:
I don't think you need the version here, if you leave it off it will pick up the version from the parent pom, which is desirable I think

Reply:
This was here to add the exclude filter file that the parent pom doesn't have

Reply:
I get that, all I'm saying is that you don't need the version stanza, you can just define the plugin and the config.